speech/grpc/src/main/java/com/examples/cloud/speech/StreamingRecognizeClient.java

```diff
@@ -76,6 +81,10 @@ public StreamingRecognizeClient(ManagedChannel channel, String file, int samplin
     this.channel = channel;
 
     speechClient = SpeechGrpc.newStub(channel);
+
+    //Send log4j logs to Console
```

Comment:
It generally is, but I thought for the purpose of sample was bit of an overkill. But I can change if folks have other thoughts.